.buildkite/pipeline.yml

```diff
@@ -0,0 +1,176 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Should we also consider BSD-3-Clause?

Reply:
I am not sure if the BSD-3-Clause is needed in this case because this is not an "exportable" repository. If we do add the BSD-3-clause license as well all crates in rust-vmm need to have a copyright and attribution for this repository which is just running the tests. I find that to be an overkill, but if people think we should add it, I'll follow.